### PR TITLE
hyva-themes/magento2-hyva-admin#2 alpine.js loaded locally to avoid CSP

### PR DESCRIPTION
**Pull request by paugnu**
Hi @Vinai and @wigman,

To solve hyva-themes/magento2-hyva-admin#2, I've:

- Imported the last version of alpine.js inside the module (2.8.0). Not minified for the moment, but I guess we should do it?
- Modified the require_js.phtml file to change the link so that it refers to the local file instead of external

I've tried doing what would be to be supposed the right way: adding it in the head block in the layout file. However, as Vinai already checked, this does not seem possible in Magento 2. Require js is loaded before the web starts breathing...

Please don't hesitate telling me if you think this could be done in a better/different way.

### CODE REVIEW

**Comment by Vinai** [comment]
@paugnu Thank you very much!
Since loading alpine is @wigman's domain I would love to get his LGTM before merging.

**Comment by wigman** [comment]
This is perfect. As long as we don't have a proper way to fix the requirejs/alpinejs conflict this is the simplest way to do it.
I'm pretty sure fixing it in another way will mean modifying the requirejs library, so let's no go there now :)

**Review by wigman — APPROVED** [review]

**Pull request merged**
